python/mxnet/contrib/onnx/mx2onnx/export_onnx.py

```diff
@@ -77,7 +78,8 @@ def register(op_name):
         """Register operators"""
         def wrapper(func):
             """Helper function to map functions"""
-            MXNetGraph.registry_[op_name] = func
+            if 'onnx' in sys.modules:
```

Comment:
a more pythonic way is to try-catch ImportError

Reply:
import onnx as _

Reply:
missed it. corrected and resubmitted. thanks.